src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -112,6 +113,11 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
   lazy val typeAliasDecl: Parser[DeclTree] =
     "type" ~> typeName ~ tparams ~ ("=" ~> typeDesc) <~ opt(";") ^^ TypeAliasDecl
 
+  lazy val importDecl: Parser[DeclTree] =
+    "import" ~ rep(not(";") ~ anyToken) ~ ";" ^^^ ImportDecl
```

Comment:
Er ... no, that does not make much sense. You're trying to lex the import away, à la regex `[^;]*;`, but that's a very fragile way to do it. You should instead truly *parse* the import statement, using its complete grammar.

Reply:
The asterisk used for wildcards `import *` is not recognized by the lexer yet so it results in a `failure("illegal character")`. It doesn't seem to fit well into any of the existing token types though. What do you think? Should we add a new one or put it into one of the existing nevertheless?

Reply:
It should be in `lexical.delimiters ++=`.

Reply:
I've changed it to parse imports properly now.